go.mod

```diff
@@ -1,15 +1,15 @@
 module github.com/armory/armory-cli
 
-go 1.19
+go 1.20
 
 require (
 	cuelang.org/go v0.4.3
 	github.com/ahmetb/go-linq/v3 v3.2.0
-	github.com/armory-io/deploy-engine v0.109.6
-	github.com/armory-io/preview-service v0.5.0
+	github.com/armory-io/deploy-engine v0.137.4-0.20230901091304-4ff4f2344d74 // TODO change to release version
```

Comment:
shall we wait with the two other pr's to be checked in and proceed with this one?